core/src/main/java/org/apache/accumulo/core/file/blockfile/cache/lru/CachedBlockQueue.java

```diff
@@ -91,24 +93,19 @@ public void add(CachedBlock cb) {
    * @return list of cached elements in descending order
    */
   public CachedBlock[] get() {
-    LinkedList<CachedBlock> blocks = new LinkedList<>();
-    while (!queue.isEmpty()) {
-      blocks.addFirst(queue.poll());
-    }
-    return blocks.toArray(new CachedBlock[blocks.size()]);
+     CachedBlock[] blocks = queue.toArray(new CachedBlock[0]);
```

Comment:
Could this be `queue.toArray(new CachedBlock[queue.size()])`?

Reply:
@belugabehr responded to my similar comment above in https://github.com/apache/accumulo/pull/1333#discussion_r317404335